en/asgardeo/docs/references/service-extensions/pre-flow-extensions/pre-update-profile-action/api/pre-update-profile-action-v1.yaml

```diff
@@ -0,0 +1,296 @@
+openapi: 3.0.1
+info:
+  title: API contract for pre update profile action
+  description: This API defines the REST API contract for a service that implements logic to extend the profile update flow of Asgardeo.
+  version: v1
+security:
+- BasicAuth: []
+- BearerAuth: []
+- ApiKeyAuth: []
+- OAuth2: []
+paths:
+  /:
```

Comment:
Why don't we define the endpoint here ? like /event

Reply:
well it's upto the implementer.
We haven't defined a path in any of the extensions as of now